app/controllers/idv/image_uploads_controller.rb

```diff
@@ -92,13 +95,15 @@ def add_costs(client_response)
     end
 
     def presenter_response(image_form_response, client_response, doc_pii_form_response)
-      form_response = image_form_response
-      if doc_pii_form_response.present?
-        form_response = image_form_response.merge(doc_pii_form_response)
+      # image form wasn't valid
+      return image_form_response unless image_form_response.success?
+
+      # doc_pii_form exists, but wasn't valid
+      if doc_pii_form_response.present? && !doc_pii_form_response.success?
+        return doc_pii_form_response.presence
```

Comment:
the `.presence` is redundant since we know from the guard outside that it is opesent?

```suggestion
      if doc_pii_form_response.present? && !doc_pii_form_response.success?
        return doc_pii_form_response
```

Reply:
doh!